pkg/kubefedctl/federate/federate.go

```diff
@@ -161,6 +161,13 @@ func NewCmdFederateResource(cmdOut io.Writer, config util.FedConfig) *cobra.Comm
 
 // Run is the implementation of the `federate resource` command.
 func (j *federateResource) Run(cmdOut io.Writer, config util.FedConfig) error {
+	var err error
+	if len(j.resourceNamespace) == 0 {
+		j.resourceNamespace, err = util.GetNamespace(j.HostClusterContext, j.Kubeconfig, config)
+	}
+	if err != nil {
+		return errors.Wrap(err, "Failed to resolve namespace")
```

Comment:
I think the error message returned by `util.GetNamespace` is sufficiently detailed that further wrapping is not suggested.

Reply:
Done